ovs/lib/helpers/generic/scrubber.py

```diff
@@ -1095,21 +1096,25 @@ def execute_scrubbing(self):
             self._logger.info('{0} - Spawning {1} stack{2}'.format(logging_start, stacks_to_spawn, '' if stacks_to_spawn == 1 else 's'))
             for stack_number in xrange(stacks_to_spawn):
                 scrub_target = self.scrub_locations[counter % len(self.scrub_locations)]
-                stack_worker = StackWorker(queue=vpool_queue,
-                                           vpool=vp,
-                                           scrub_info=scrub_target,
-                                           error_messages=self.error_messages,
-                                           worker_contexts=self.worker_contexts,
-                                           stack_work_handler=stack_work_handler,
-                                           job_id=self.job_id,
-                                           stacks_to_spawn=stacks_to_spawn,
-                                           stack_number=stack_number)
-                self.stack_workers.append(stack_worker)
-                stack = Thread(target=stack_worker.deploy_stack_and_scrub,
+                try:
+                    stack_worker = StackWorker(queue=vpool_queue,
+                                               vpool=vp,
+                                               scrub_info=scrub_target,
+                                               error_messages=self.error_messages,
+                                               worker_contexts=self.worker_contexts,
+                                               stack_work_handler=stack_work_handler,
+                                               job_id=self.job_id,
+                                               stacks_to_spawn=stacks_to_spawn,
+                                               stack_number=stack_number)
+                    self.stack_workers.append(stack_worker)
```

Comment:
Try except only required when instantiating the StackWorker class.
On except: log and continue